Packs/GSuiteAdmin/Integrations/GSuiteAdmin/GSuiteAdmin.py

```diff
@@ -1484,6 +1493,397 @@ def gsuite_chromeos_device_action_command(client: Client, args: dict[str, str])
+        'pageToken': page_t,
+        'maxResults': page_s
+    })
+
+
+def chromebrowser_list_command(client: Client, args: dict[str, str]) -> CommandResults:
+    '''
+        List chromebrowsers devices
+
+        :param client: Client object.
+        :param args: Command arguments - customer_id (reqyired), device_id, order_by, org_unit_path, projection, query,
+         sort_order, page_size and limit
+        :return: Command Result.
+    '''
+    API_LIMIT = 100
+    client.set_authorized_http(scopes=SCOPES['CHROME_BROWSERS'])
+    customer_id = args.get('customer_id', '')
+    device_id = args.get('device_id', '')
+    order_by = args.get('order_by', '')
+    org_unit_path = args.get('org_unit_path', '')
+    projection = args.get('projection', '')
+    query = args.get('query', '')
+    sort_order = args.get('sort_order', '')
+    page_size = args.get('page_size', '')
+    page_token = args.get('page_token', '')
+    limit = args.get('limit', '')
+
+    full_url = f'https://www.googleapis.com/admin/directory/v1.1beta1/customer/{customer_id}/devices/chromebrowsers/'
+    cb_list_resp = []
+    if device_id:
+        full_url = full_url + f'{device_id}'
+        params_for_command = GSuiteClient.remove_empty_entities({
+            'projection': projection
+        })
+        response = client.http_request(full_url=full_url, method='GET', params=params_for_command)
+        cb_list_resp.append(response)
+    else:
+        params_for_command = assign_params_chromebrowser_list(projection, query, order_by, sort_order, org_unit_path, page_token,
+                                                              page_size)
+        if limit:
+            if int(limit) <= API_LIMIT:
+                page_size = limit
+                params_for_command = assign_params_chromebrowser_list(projection, query, order_by, sort_order, org_unit_path,
+                                                                      page_token, page_size)
+                response = client.http_request(full_url=full_url, method='GET', params=params_for_command)
+                cb_list_resp.extend(response['browsers'])
+            else:
+                while len(cb_list_resp) < int(limit):
+                    if int(limit) - len(cb_list_resp) > API_LIMIT:
+                        page_size = str(API_LIMIT)
+                    else:
+                        page_size = str(int(limit) - len(cb_list_resp))
+                    params_for_command = assign_params_chromebrowser_list(projection, query, order_by, sort_order, org_unit_path,
+                                                                          page_token, page_size)
+                    response = client.http_request(full_url=full_url, method='GET', params=params_for_command)
+                    page_token = response.get('nextPageToken', '')
+                    cb_list_resp.extend(response['browsers'])
+                    if not page_token:
+                        break
+        else:
```

Comment:
I thought about it when developing.

There is a default limit in the google API - 100.
That's why I didn't want to set another default, since I though it's redundant.

What do you think?